app/google-services.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
This file should _NOT_ be added to VCS. It should be added to the Github Secrets (maybe as an encrypted base64 string), and then decoded in a step before building the app.

Reply:
Updated @AVI5HEK